Rakefile

```diff
@@ -40,4 +40,4 @@ task :console do
   IRB.start
 end
 
-task default: %i{spec style}
+task default: %i{spec}
```

Comment:
Instead of modifying the default task here it would be better to just change the expeditor config to use `rake spec`.